plugins/statsd/statsd_test.go

```diff
@@ -326,6 +326,46 @@ func TestParse_MeasurementsWithSameName(t *testing.T) {
 	}
 }
 
+// Test that measurements with multiple bits, are treated as different outputs
+func TestParse_MeasurementsWithMultipleValues(t *testing.T) {
+	s := NewStatsd()
+
+	// Test that counters work
+	valid_lines := []string{
+		"valid.multiple:0|ms|@0.1:1|ms",
```

Comment:
Can you add more options here? I'd like to see:

without sample rate:
```
"valid.multiple.nosample:0|ms|:1|ms:200|ms"
```

Duplicate measurements (also, document what would happen here):
```
"valid.multiple.duplicate:0|ms:1|ms:0|ms:1|ms"
```

Duplicate measurements with different types (documentation needed here too):
```
"valid.multiple.duplicate:1|c:1|c:2|c:1|c"
"valid.multiple.duplicate:1|h:1|h:2|h:1|h"
"valid.multiple.duplicate:1|s:1|s:2|s:1|s"
"valid.multiple.duplicate:1|g:1|g:2|g:1|g" <-- what happens here? gauge will be set to the final value?
```

Multiple measurement types (is this supported?)
```
"valid.multiple.duplicate:1|c:1|ms:2|s:1|g"
```